biome.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.7.3/schema.json",
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "lineWidth": 100,
+    "attributePosition": "auto"
+  },
+  "organizeImports": {
+    "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "style": {
+        "noParameterAssign": "off",
```

Comment:
Should this be a `warn`? Parameter reassignment could lead to issues

Reply:
Good idea! I would like to fully enable this rule, but we violate it at many places and I prefer tackling them in a separate PR.